src/ports/web/api/duplicates.rs

```diff
@@ -0,0 +1,51 @@
+use super::Result;
+use crate::{
+    core::{
+        prelude::*,
+        usecases::{self, DuplicateType},
+        util,
+    },
+    infrastructure::db::sqlite,
+};
+use rocket_contrib::json::Json;
+
+#[post("/duplicates/check-place", data = "<body>")]
+pub fn post_duplicates(
+    db: sqlite::Connections,
+    body: Json<usecases::NewPlace>,
+) -> Result<Vec<(String, DuplicateType)>> {
+    let all_entries = {
```

Comment:
We should query only entries within a surrounding bounding box around each given entry from the search engine and use these results for the duplicate check. Please add a TODO or FIXME when adding code that obviously doesn't scale well so that we are able to track open issues.